android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -390,7 +390,11 @@ private void sendAdLoadedReactNativeEventForAd(final MaxNativeAd ad)
         nativeAdInfo.putString( "advertiser", ad.getAdvertiser() );
         nativeAdInfo.putString( "body", ad.getBody() );
         nativeAdInfo.putString( "callToAction", ad.getCallToAction() );
-        nativeAdInfo.putDouble( "starRating", ad.getStarRating() );
+
+        if ( ad.getStarRating() != null )
+        {
+            nativeAdInfo.putDouble( "starRating", ad.getStarRating().doubleValue() );
```

Comment:
Isn't behaviour different on both platforms now? On iOS, the nil `NSNumber` will get converted to `0` (there's nos uch thing as NPE in iOS)

Reply:
Sounds good